src/SDKs/ApplicationInsights/DataPlane/ApplicationInsights/Microsoft.Azure.ApplicationInsights.Query.csproj

```diff
@@ -5,15 +5,17 @@
   
   <PropertyGroup>
     <Description>Microsoft Azure Application Insights Data Library</Description>
-    <VersionPrefix>0.9.0.0-preview</VersionPrefix>
-    <AssemblyName>Microsoft.Azure.ApplicationInsights</AssemblyName>
-    <PackageId>Microsoft.Azure.ApplicationInsights</PackageId>
-    <PackageTags>Management.ApplicationInsights;</PackageTags>
+    <VersionPrefix>1.0.0.0</VersionPrefix>
+    <VersionSuffix>Preview-1</VersionSuffix>
```

Comment:
Please remove this
Also, is the package stable or preview?
If this is still preview, the version can be bumped to `0.10.0-preview` instead